.storybook/addons/codeEditorAddon/codeAddon.js

```diff
@@ -73,21 +73,34 @@ export const withCodeEditor = makeDecorator({
       storyElement = story;
     }
 
+    // format a string to fit editor indentation
+    // index is the position of first char after spaces (number of spaces from beginning of block)
+    // when writing stories in our current format, there are always 2 spaces for html, 4 spaces for css and js
```

Comment:
why different spacing for different formats - shouldn't they all be the same spacing?

Reply:
cause script and styles are nested in another layer down with one more tab...

Reply:
I see. We should avoid depending on the current code formatting of the stories and format the code programmatically from how it was written in the stories. Otherwise we get locked in to a specific formatting that makes things difficult to change down the line.